src/app/services/auth.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class AuthService {
+  constructor() {}
+
+  public isAuthenticated(): boolean {
+    return !!localStorage.getItem('token');
```

Comment:
IMHO `localStorage` should only be used for persistence of state not called directly 
![image](https://github.com/Titi999/sef-foundation-frontend/assets/57860381/51d716e1-063e-430f-abcc-b8814feaa9fa)
We can do something like so ensure it is also reactive